cclib/method/ddec.py

```diff
@@ -232,18 +242,18 @@ def calculate(self, indices=None, fupdate=0.05):
             self.logger.info("Using charge densities from the provided Volume object.")
             self.chgdensity = self.volume
 
-        # STEP 1
+        # * STEP 1 *
         # Carry out step 1 of DDEC6 algorithm [Determining ion charge value]
         # Refer to equations 49-57 in doi: 10.1039/c6ra04656h
-        self.logger.info("Creating first reference charges.")
+        self.logger.info("Creating first reference charges. (Step 1/7)")
         ref, loc, stock = self.calculate_refcharges()
```

Comment:
The comment above says "determining ion charge value" but here we see "ref", "loc" and "stock". It would be good to make it clear which one(s) "ion charge value" refers to.

Also, prefer longer, more descriptive names: "reference_charges", "stockholder", etc. I know these are temporary, but it's good to have this habit in Python.

Reply:
I've changed them into longer names. Thanks.